priv/README

```diff
@@ -0,0 +1 @@
+too lazy to detect and create empty priv folder from makefile ;)
```

Comment:
May be now is a good time to do that 😉

Reply:
Please let me if we cann't create missing folders from Windows Makefiles and why?

Reply:
mkdir in Makefile -- solved